common4j/build.gradle

```diff
@@ -165,6 +166,10 @@ if (project.hasProperty("dc")) {
     dcParameter = dc
 }
 
+if (project.hasProperty("useRealAuthority")) {
```

Comment:
Please add comments describing what this is doing because otherwise I have no idea. Also please add a PR description

Reply:
Addressed

Reply:
This is specific to the Native Auth work, right? So let's use a specific variable/parameter name here.. (realAuthority is just too generic)..

Also you should explore if you can achieve this without build config values. If you had used Robolectric for instance then you wouldn't need to bake this logic into build config and you would've been able to set this for testing only more easily.

Reply:
Addressed